projects/whats-cookin-part-one.md

```diff
@@ -265,3 +241,37 @@ You are *not expected* to test:
   * Overview of technologies used, challenges, wins, and any other reflections
   * Screenshots/gifs of your app
   * List of contributors
+* **Team productively and professionally works through challenges and conflicts to ensure all team members are able to be heard and contribute throughout the project.**
+  * Instructors are available to offer support and guidance but conversations around what *is* and what *is not* working are expected to be led by the team members themselves.
+
+### Fundamental JavaScript & Style / OOP / Fetch
+- Code is divided into logical components each with a clean, single responsibility
+- Methods use array and object prototypes, not for loops
+- DOM manipulation is not present in class components
+- Variables and methods are well named
+- Code leverages JavaScript's truthy/falsey principles
+
+### Test-Driven Development
+- Application has a robust and thorough test suite that covers all class methods and class properties.
+- Test suite is organized.
+  - Each class method/class property is tested in it's own it block.
```

Comment:
This line should not have the apostrophe in the `it's`.

Reply:
Same goes for lines below.

Reply:
Fixing - thank you!